<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Axes</class>
 <widget class="QWidget" name="Axes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>217</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_AxixName">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="3,3,4,2,2,2" columnstretch="2,1,3,4,4,2,2,5">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="3" column="0">
       <widget class="QComboBox" name="comboBox_I2cAddress">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="text_i2cAddress">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>I2C address</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QSpinBox" name="spinBox_ChanelEncoder">
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLabel" name="text_prescaler">
        <property name="text">
         <string>Prescaler %</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_Output">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Output</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="5" colspan="2">
       <widget class="QComboBox" name="comboBox_AxisSource2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_Inverted">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Inverted</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5" colspan="2">
       <widget class="QLabel" name="text_axisSource2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Axis source 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QLabel" name="text_stepDiv">
        <property name="text">
         <string>Step div</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="3" rowspan="2">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_Button2">
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="text_button2">
           <property name="text">
            <string>Button 2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QComboBox" name="comboBox_Button2"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2" colspan="5">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QProgressBar" name="progressBar_Out">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QProgressBar::chunk {
     background-color: rgb(139, 0, 0);
 }
QProgressBar::chunk:disabled {
     background-color: rgba(139, 0, 0, 80);;
 }</string>
          </property>
          <property name="minimum">
           <number>-32767</number>
          </property>
          <property name="maximum">
           <number>32767</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string notr="true">%v</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar_Raw">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QProgressBar::chunk {
     background-color:rgb(85, 107, 47);
 }
QProgressBar::chunk:disabled {
     background-color: rgba(85, 107, 47, 80);;
 }</string>
          </property>
          <property name="minimum">
           <number>-32767</number>
          </property>
          <property name="maximum">
           <number>32767</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string notr="true">%v</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QLabel" name="text_function">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Function</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QSpinBox" name="spinBox_Prescaler">
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="text_chanelEncoder">
        <property name="text">
         <string>Chanel/Encoder</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="7" rowspan="6">
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,1,0,1" columnminimumwidth="0,5,0,0,5,0">
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="4" column="2" colspan="2">
          <widget class="QPushButton" name="pushButton_SetCenter">
           <property name="text">
            <string>Set Center</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="text_maximum">
           <property name="text">
            <string>Maximum</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="spinBox_Offset">
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="QSpinBox" name="spinBox_Deadband">
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum">
            <number>127</number>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="text_minimum">
           <property name="text">
            <string>Minimum</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="text_bits">
           <property name="text">
            <string> bits</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2" colspan="2">
          <widget class="QSpinBox" name="spinBox_CalibMin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>-32767</number>
           </property>
           <property name="maximum">
            <number>32767</number>
           </property>
           <property name="value">
            <number>-32767</number>
           </property>
          </widget>
         </item>
         <item row="6" column="2" colspan="2">
          <widget class="QSpinBox" name="spinBox_CalibCenter">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>-32767</number>
           </property>
           <property name="maximum">
            <number>32767</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButton_StartCalib">
           <property name="text">
            <string>Calibrate</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="text_deadband">
           <property name="text">
            <string>Deadband</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2" colspan="2">
          <widget class="QSpinBox" name="spinBox_CalibMax">
           <property name="minimum">
            <number>-32767</number>
           </property>
           <property name="maximum">
            <number>32767</number>
           </property>
           <property name="value">
            <number>32767</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="spinBox_Resolution">
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>16</number>
           </property>
           <property name="value">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="text_resolution">
           <property name="text">
            <string>Resolution</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="checkBox_Center">
           <property name="text">
            <string>Center</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="text_offset">
           <property name="text">
            <string>Offset</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QCheckBox" name="checkBox_DynDeadband">
           <property name="text">
            <string>Dynamic deadband</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="text_angle">
           <property name="text">
            <string notr="true"> Â°</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="text_filter">
           <property name="text">
            <string>Filter</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="0" colspan="4">
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Reset calibration</string>
           </property>
          </widget>
         </item>
         <item row="8" column="5">
          <widget class="QLabel" name="label_Filter">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Off</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="5" rowspan="7">
          <widget class="QSlider" name="sliderV_Filter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>7</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBothSides</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="3" colspan="2">
       <widget class="QComboBox" name="comboBox_Function"/>
      </item>
      <item row="4" column="4" rowspan="2">
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="text_button3">
           <property name="text">
            <string>Button 3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_Button3">
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QComboBox" name="comboBox_Button3"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QSpinBox" name="spinBox_StepDiv">
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Out</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Raw</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_AxisSource1"/>
      </item>
      <item row="4" column="1" rowspan="2" colspan="2">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="text_button1">
           <property name="text">
            <string>Button 1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_Button1">
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QComboBox" name="comboBox_Button1">
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="text_axisSource1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Axis source 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
